packages/ui/src/components/Organisms/ContentHub.tsx

```diff
@@ -70,41 +70,51 @@ export function ContentHub({ pageSize = 10 }: { pageSize: number }) {
   );
 }
 
-export const Card = (item: ContentHubResultItemFragment) => (
-  <article className="focus-within:outline focus-within:outline-2 focus-within:outline-offset-2 focus-within:outline-indigo-600 relative max-w-sm bg-white rounded-lg hover:shadow dark:bg-gray-800 dark:border-gray-700 overflow-hidden">
-    <div className="rounded-t-lg">
-      {item.teaserImage ? (
-        <Image {...item.teaserImage} className="w-full" />
-      ) : (
-        <div className="aspect-[4/3] bg-indigo-200" />
-      )}
-    </div>
-    <div className="p-5">
-      <h5 className="mb-2 text-2xl font-bold tracking-tight text-gray-900 dark:text-white">
-        {item.title}
-      </h5>
-      <Link
-        href={item.path}
-        className="inline-flex items-center px-3 py-2 text-sm font-medium text-center text-blue-700 border border-blue-700 rounded-lg hover:bg-blue-800 hover:text-white focus:outline-offset-4 dark:bg-blue-600 dark:hover:bg-blue-700 dark:focus:ring-blue-800 after:content-[''] after:absolute after:inset-0"
-      >
-        <span className="invisible w-0 h-0 overflow-hidden">{item.title} </span>
-        Read more
-        <svg
-          className="rtl:rotate-180 w-3.5 h-3.5 ms-2"
-          aria-hidden="true"
-          xmlns="http://www.w3.org/2000/svg"
-          fill="none"
-          viewBox="0 0 14 10"
+const Card = (item: ContentHubResultItemFragment) => {
+  const id = item.title.toLowerCase().replace(/\s+/g, '-');
```

Comment:
I would not use the title for the ID, but get it from the ID of the map function and inject into the component.

As the regex is only replacing spaces, so `A title with "quotes" or exclamation! marks` will end up like `a-title-with-"quotes"-or-exclamation!-marks`